server/map_rpc_server.go

```diff
@@ -180,6 +180,11 @@ func (t *TrillianMapServer) GetLeavesByRevisionNoProof(ctx context.Context, req
 		return nil, err
 	}
 
+	// Remove LeafHash because SetLeaves does not supply it.
+	for _, l := range leaves {
```

Comment:
Please update the CL description to explicitly call out this change in functional behaviour (is this also worth updating changelog for?).